api/serializers.py

```diff
@@ -12,6 +12,15 @@
 class ProfileSerializer(serializers.ModelSerializer):
     def to_representation(self, instance):
         rep = super().to_representation(instance)
+
+        # Replace image file path with the file data.
+        image = instance.image
+        if image != "":
```

Comment:
```suggestion
        if hasattr(image, "file"):
```
The existing implementation returns an internal server error if the instance does not have an associated file for the image.